buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -46,12 +46,56 @@ import java.util.jar.Attributes
 
 
 enum class ModuleType(val compatibility: JavaVersion) {
+
+    /**
+     * This module type is used internally by default but should not be
+     * referenced by any module.
+     */
     UNDEFINED(JavaVersion.VERSION_1_1),
+
+    /**
+     * This module type is used by modules that contain code that is used by one
+     * of the entry points of using Gradle, such as the Wrapper, the Launcher
+     * and the Tooling API.
```

Comment:
You might want to use `<p>` tags for the paragraphs if that's your intention.

Reply:
Or maybe the behaviour is different with Kotlin source. And of course, if this isn't part of the public API, how the text appears in the online docs is probably irrelevant.

Reply:
Yeah, we don't generate docs for this as it is only used internally. Thus, I deliberately left out `<p>` tags. 🙂